src/components/area/panel/panelList.tsx

```diff
@@ -12,6 +12,16 @@ interface PanelListProps {
 }
 
 export default function PanelList (props: PanelListProps): JSX.Element {
+  const [favs, setFavs] = useState<string[]>([])
+  useEffect(() => {
+    fetch('/api/user/fav')
+      .then(async res => await res.json())
+      .then(collections => {
+        setFavs(collections.areaCollections.favourites)
```

Comment:
I think `favs` is undefined here (for users who have added areas to their favorite) causing a blank screen.

<img width="847" alt="Screen Shot 2022-07-31 at 4 16 00 PM" src="https://user-images.githubusercontent.com/3805254/182030624-b274e1d8-43fd-4781-89c0-3ae6a6e08a2e.png">

To reproduce:

1.  Log in Auth0 and remove user's metadata `collections`
2. Visit an area page such as https://open-tacos-dbdv8vk67-openbeta-dev.vercel.app/areas/47667307-c0a0-5f50-873d-2a5c6d757fb2